ArduCopter/FlightMode.h

```diff
@@ -57,7 +56,7 @@ class FlightMode {
     AC_WPNav *&wp_nav;
     AC_PosControl *&pos_control;
     AP_InertialNav &inertial_nav;
-    AP_AHRS &ahrs;
+    AP_AHRS_NavEKF &ahrs;
```

Comment:
Yes, but it actually does get passed an AHRS_EKF in Copter.

This change was required as part of moving the call which fills in the optical flow scalars into here.

I've already tried to point out that that call shouldn't be made how it is ATM; while cheap in the non-OF case, it should be called at all if we're never going to use the values (PosControl seems to run at 50Hz, we calculate at 400Hz, and e.g. ACRO will never use the values!)